roles/nodejs/molecule/default/verify.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: Verify
+  hosts: all
+  gather_facts: false
+  tasks:
+  - name: check nodejs package status
+    package:
+      name: "{{ item }}"
+      state: present
+    check_mode: true
+    register: pkg_status
```

Comment:
hey @carolyncole I was working on this assumption. When the loop runs everything is registered as pkg_status. If any item is missing the assertion will fail. 

```
(princeton_ansible) kayiwa@PU-F5KSY01XF9VN ~/D/d/p/p/r/nodejs (1851_move_nodejs)> molecule verify
--> Test matrix

└── default
    └── verify

--> Scenario: 'default'
--> Action: 'verify'
--> Running Ansible Verifier
--> Sanity checks: 'docker'

    PLAY [Verify] ******************************************************************

    TASK [check nodejs package status] *********************************************
    ok: [instance] => (item=nodejs)
    ok: [instance] => (item=yarn)
    changed: [instance] => (item=netcat)

    TASK [test for nodejs packages] ************************************************
fatal: [instance]: FAILED! => {
    "assertion": "not pkg_status.changed",
    "changed": false,
    "evaluated_to": false,
    "msg": "Assertion failed"
}

    PLAY RECAP *********************************************************************
    instance                   : ok=1    changed=1    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0

ERROR:
```